Dockerfile

```diff
@@ -0,0 +1,81 @@
+# base image
+FROM ruby:2.3.0
+
+# set environment variables
+ENV \
+  APP_HOME=/legislated \
+  PG_VERISON=9.6 \
+  PHANTOM_JS_VERSION=phantomjs-2.1.1-linux-x86_64
+
+RUN \
+  # logging which commands actually runs
+  set -x && \
```

Comment:
I like to use -e as well but that is just a prefernce.

Reply:
i'm not super familiar with `set -e`, but since everything is `&&`'d here, will it do anything?